README.md

```diff
@@ -1 +1,45 @@
-# hw01
+# Emily Young - HW01  
```

Comment:
This is a really detailed intro of yourself and I really like it! It's comfortable to read when you make some important terms in bold throughout the bio.